framework/base/Security.php

```diff
@@ -605,19 +470,8 @@ public function generatePasswordHash($password, $cost = null)
             $cost = $this->passwordHashCost;
         }
 
-        if (function_exists('password_hash')) {
-            /* @noinspection PhpUndefinedConstantInspection */
-            return password_hash($password, PASSWORD_DEFAULT, ['cost' => $cost]);
-        }
-
-        $salt = $this->generateSalt($cost);
-        $hash = crypt($password, $salt);
-        // strlen() is safe since crypt() returns only ascii
-        if (!is_string($hash) || strlen($hash) !== 60) {
-            throw new Exception('Unknown error occurred while generating hash.');
-        }
-
-        return $hash;
+        /* @noinspection PhpUndefinedConstantInspection */
```

Comment:
inspection annotation is not needed as the constant should be there in any version of PHP >=7.1